keysManagement/keysHolder.go

```diff
@@ -67,6 +75,28 @@ func checkVirtualPeersHolderArgs(args ArgsVirtualPeersHolder) error {
 	return nil
 }
 
+func createDataMap(namedIdentities []config.NamedIdentity) map[string]*peerInfo {
+	dataMap := make(map[string]*peerInfo)
+
+	for _, identity := range namedIdentities {
+		for _, blsKey := range identity.BLSKeys {
+			bls, err := hex.DecodeString(blsKey)
+			if err != nil {
+				continue
```

Comment:
maybe return error here? The node should stop prematurely if the config is not done properly